lib/palindrome_check.rb

```diff
@@ -1,7 +1,40 @@
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) where n is the lenght of my_phrase
+# Space complexity: O(1) because no additional space is needed
 def palindrome_check(my_phrase)
-  raise NotImplementedError
+  return false if my_phrase.nil?
```

Comment:
What should your method return if `my_phrase == ""`? Is that a valid case considering `"" != nil`?